calc.js

```diff
@@ -0,0 +1,73 @@
+spinnerHide();
+let x = 0;
```

Comment:
why declare x & y here? you are using them only inside 1 function, so they can be created there. no need for global scoping.